src/ValueObject/Column.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\TelemetryData\ValueObject;
+
+class Column
```

Comment:
Pride mi to trocha duplicitne, kedze mame `db-extractor-table-format`, ktory sa da vyuzit samostatne.
Chapem, ze si chcel to parsovanie `length` dat do `datatype` kniznice a pouzivat tam `array`, ... no vo vysledku si tu musel nanovo vytvorit tieto value objects, ktore maju o dost viac riadkov ako spracovanie tej `length`. 

Okrem toho je `mysql` specificka v tom, ze ma implementovany typ  `datatype` kniznici, pri inych backendoch by sa to aj tak riesilo v "lokalne" alebo cez `Generic` typ.

Idealne by bolo asi upravit `db-extractor-table-format` aby mohla byt `length` string alebo array.

Samozrejme nechcem aby si to prepisoval, iba poznamka do buducna.